front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx

```diff
@@ -158,110 +158,110 @@ const ScenarioV2 = () => {
       <main className="mastcontainer mastcontainer-no-mastnav">
         <div className="scenario">
           <div className="row scenario-container">
-            <div className={collapsedTimetable ? 'd-none' : 'col-hdp-3 col-xl-4 col-lg-5 col-md-6'}>
-              <div className="scenario-sidemenu">
```

Comment:
Note for reviewers: this big diff is caused by merging together these two `<div>` elements. There is no change below apart from removing one indent level.

Reply:
For those who don't know, in the ⚙️ above, the following option might help

![image](https://github.com/OpenRailAssociation/osrd/assets/2520723/68450a2f-d1b0-4a58-ae96-0394847ceb49)